{{cookiecutter.project_slug}}/.github/workflows/expo-pr.yml

```diff
@@ -16,6 +15,7 @@ jobs:
       PR: {{ "${{ env.PR }}" }}
 
   publish:
+    if: github.event.deployment_status.state == 'success' # guarantee backend deployment first
```

Comment:
why move this?

Reply:
@oudeismetis is it going to cause a problem? I moved it because the successful deployment does not need to occur to find the backend PR. 
I guess leaving it in there also avoids running that one as well